addons/website_forum/static/src/xml/website_forum_share_templates.xml

```diff
@@ -72,4 +72,17 @@
             </div>
         </div>
     </t>
+    <t t-name="website_forum.web_editor_toolbar" t-extend="web_editor.toolbar">
```

Comment:
Create an extra commit in this PR to rename the `website_forum_share_templates.xml` file to something like `public_templates.xml` (it's not only about share anymore and I personally don't like repeating "website_forum" in file names, good opportunity to get rid of it.

(and the other website_forum_templates.xml we don't care about as we will get rid of it with the frontend to backend task anyway)